### PR TITLE
Commissar Regiment Selection

### PR DESCRIPTION
**Pull request by Max-023**
Gives Commissars access to a regiment selection. Available regiments are Cadians (same uniform), Kriegers, and Catachans. Removes the Commissar mask from the office, as well as the chainsword (Catachan Comm gets it).

### CODE REVIEW

File: code/modules/clothing/shoes/jobs.dm [hunk 1]
```diff
@@ -141,6 +141,13 @@
 	icon_state = "kriegboots"
 	item_state = "kriegboots"
 
+/obj/item/clothing/shoes/commissar
+	name = "boots of the commissar"
```
Comment (EliteRiceFarmer) [inline code/modules/clothing/shoes/jobs.dm:145]: Commissariat Boots

**Review by EliteRiceFarmer — CHANGES_REQUESTED** [review]

File: code/modules/clothing/shoes/jobs.dm (hunk 1 above)
Comment (EliteRiceFarmer) [inline code/modules/clothing/shoes/jobs.dm:145]:
```suggestion
	name = "Commissariat Service Boots"
```

File: code/modules/clothing/shoes/jobs.dm [hunk 2]
```diff
@@ -141,6 +141,13 @@
...
+	desc = "Knee high riding boots of an officer of the Officio Prefectus."
```
Comment (EliteRiceFarmer) [inline code/modules/clothing/shoes/jobs.dm:146]:
```suggestion
	desc = "Service Boots issued to Commissar of the Officio Prefectus."
```

File: code/modules/clothing/suits/armor.dm [hunk 3]
```diff
@@ -599,3 +599,15 @@
 /obj/item/clothing/suit/armor/commissar/New()
 	..()
 	slowdown_per_slot[slot_wear_suit] = 0.5
+
+/obj/item/clothing/suit/armor/commissar/krieg
+	name = "Commissar's Trenchcoat"
+	desc = "The Commissar's magnificent, charismatic that strikes dreads and fears into the mind of lowly Imperial Guardsmen and heretics alike. Plated in Caraspace, giving it much more protection than average Guardsman."
```
Comment (EliteRiceFarmer) [inline code/modules/clothing/suits/armor.dm:605]:
```suggestion
	desc = "The Krieg Commissar's Trenchcoat, Worn and damaged by flame of war, Plated in Caraspace for extra protection and decorated in Krieg style, golden-lining span across the coat, Recreating the Emperor's Light."
```

File: code/modules/clothing/under/jobs/security.dm [hunk 4]
```diff
@@ -242,3 +242,10 @@
 	icon_state = "camopants"
 	cold_protection = LOWER_TORSO
 	min_cold_protection_temperature = SPACE_SUIT_MIN_COLD_PROTECTION_TEMPERATURE
+
+/obj/item/clothing/under/rank/commissar
+	name = "krieg Battle Uniform"
```
Comment (EliteRiceFarmer) [inline code/modules/clothing/under/jobs/security.dm:247]:
```suggestion
	name = "Commissariat Dress Uniform"
```

File: code/modules/clothing/under/jobs/security.dm [hunk 5]
```diff
@@ -242,3 +242,10 @@
...
+	desc = "What was once a full green uniform has been stripped down for survivability on the inhospitable world of catachan, though now it just seems like a power move to wear this on a frozen tundra."
```
Comment (EliteRiceFarmer) [inline code/modules/clothing/under/jobs/security.dm:248]:
```suggestion
	desc = "Under the rigorous trenchcoat of an Commissar is his neatly made service uniform, featuring his nametag and medals on their set of collar dress shirt and service pants."
```

**Review by EliteRiceFarmer — CHANGES_REQUESTED** [review]

File: code/modules/clothing/under/jobs/security.dm (hunk 5 above)
Comment (EliteRiceFarmer) [reply code/modules/clothing/under/jobs/security.dm:248]: I don't know if this is for catachan only or for all commissar. If not, don't merge.

We also need undershirt for normal commissar as well, so make one if you can.

**Comment by Max-023** [comment]
closing due to revisions in the discord

**Pull request closed without merging**